lib/Server.js

```diff
@@ -155,6 +155,10 @@ class Server {
       }
 
       this.sockWrite(this.sockets, 'progress-update', { percent, msg });
+
+      if (this.listeningApp) {
+        this.listeningApp.emit('progress-update', { percent, msg });
+      }
```

Comment:
Why we need check `this.listeningApp`? Is can be undefined?

Reply:
We need this condition because this.setupProgressPlugin is called before this.createServer.